.travis.yml

```diff
@@ -32,13 +50,19 @@ cache:
     - $HOME/.ccache
 
 before_install:
-  - ccache -s
   - mkdir -p "${HOME}/.cache/download"
-  - ci/install-edm.sh
-  - export PATH="${HOME}/edm/bin:/usr/lib/ccache:${PATH}"
-  - edm install -y wheel click coverage
   - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
+  - if [[ ${TRAVIS_OS_NAME} == "linux" ]] ; then
+       ccache -s;
+       sh -e /etc/init.d/xvfb start;
+       ./ci/install-edm-linux.sh;
+       export PATH="${HOME}/edm/bin:${PATH}";
+    else
+       ./ci/install-edm-osx.sh;
+       export PATH="${PATH}:/usr/local/bin";
+    fi
+  - edm install -y wheel click coverage
+  - export PATH="${HOME}/edm/bin:/usr/lib/ccache:${PATH}"
```

Comment:
We do need to do this for osx correct? Also I think that any path manipulations should happen before we call edm

Reply:
moved the `edm` command after path manipulation.

Reply:
We still do not need to perform this specific manipulation for osx so the path manipulation can be consolidated with the related line in the linux branch

Reply:
done